### ISSUE TITLE
Can't use images-browser

### ISSUE THREAD

**Issue by bluenyx** [AlUlkesh/stable-diffusion-webui-images-browser#125]
IndexError: list index out of range
Traceback (most recent call last):
  File "/var/www/html/stable-diffusion-webui/venv/lib/python3.10/site-packages/PIL/JpegImagePlugin.py", line 643, in _save
   rawmode = RAWMODE[im.mode]
 KeyError: 'RGBA'
The above exception was the direct cause of the following exception:
Traceback (most recent call last):   
File "/var/www/html/stable-diffusion-webui/extensions/stable-diffusion-webui-images-browser/scripts/image_browser.py", line 811, in get_image_page
    thumbnail_list = get_image_thumbnail(image_list)
   File "/var/www/html/stable-diffusion-webui/extensions/stable-diffusion-webui-images-browser/scripts/image_browser.py", line 784, in get_image_thumbnail
    thumbnail.save(cache_image_path, "JPEG")
  File "/var/www/html/stable-diffusion-webui/venv/lib/python3.10/site-packages/PIL/Image.py", line 2431, in save
     save_handler(self, fp, filename)
   File "/var/www/html/stable-diffusion-webui/venv/lib/python3.10/site-packages/PIL/JpegImagePlugin.py", line 646, in _save
:     raise OSError(msg) from e
 OSError: cannot write mode RGBA as JPEG

**Comment by AlUlkesh** [comment]
Can you check if this fixes your issue?

**Comment by bluenyx** [comment]
This issue has been fixed. However, there is another issue. After you create the image, you can't check it in the image browser immediately, but you have to do Reload UI to check the information. It doesn't matter if it's intended.
Thank you

**Comment by AlUlkesh** [comment]
Can you please check if this change solved any of your issues: #126?

**Comment by bluenyx** [comment]
The synchronization problem seems to have been solved. Some image information errors occur, but it does not appear to be able to retrieve existing data values.

I use gradio: 3.23.0 

![image](https://user-images.githubusercontent.com/96876/230610526-d34b94bb-8aa4-479a-81eb-2cd2d3327590.png)

**Comment by AlUlkesh** [comment]
Did you completely stop and restart webui after the update? Any messages on the javascript console?

**Comment by AlUlkesh** [comment]
Can you try the latest update please: #135

**Comment by bluenyx** [comment]
I did an update today and it fixed the issue.

The problem was solved elsewhere, in my case by using Nginx to access the WebUI.

I checked the Nginx log and found the following error.

[error] 29582#29582: *3986 client intended to send too large body: 1418615 bytes, 

So I added the http option to nginx.conf and no more errors.

http {
...
 client_max_body_size 15m;
...
}

Fixed problem. close this issue
Thank you
